lntest/itest/lnd_test.go

```diff
@@ -14252,34 +14252,50 @@ func TestLightningNetworkDaemon(t *testing.T) {
 	aliceBobArgs := []string{
 		"--default-remote-max-htlcs=483",
 	}
-	if err = lndHarness.SetUp(aliceBobArgs); err != nil {
-		ht.Fatalf("unable to set up test lightning network: %v", err)
-	}
 
 	// Run the subset of the test cases selected in this tranche.
 	for idx, testCase := range testCases {
 		testCase := testCase
-		logLine := fmt.Sprintf("STARTING ============ %v ============\n",
-			testCase.name)
+		name := fmt.Sprintf("%02d-of-%d/%s/%s",
+			trancheOffset+uint(idx)+1, len(allTestCases),
+			chainBackend.Name(), testCase.name)
 
-		err := lndHarness.EnsureConnected(
-			context.Background(), lndHarness.Alice, lndHarness.Bob,
-		)
-		require.NoError(t, err, "unable to connect alice to bob")
+		success := t.Run(name, func(t1 *testing.T) {
+			cleanTestCaseName := strings.ReplaceAll(
+				testCase.name, " ", "_",
+			)
 
-		err = lndHarness.Alice.AddToLog(logLine)
-		require.NoError(t, err, "unable to add to Alice's log")
+			err = lndHarness.SetUp(cleanTestCaseName, aliceBobArgs)
+			if err != nil {
+				ht.Fatalf("unable to set up test lightning "+
```

Comment:
This must also call `t1.Fatalf` to avoid the "subtest called fail" error.

Reply:
Yes, indeed. Done.